.github/workflows/create-update-issues.yaml

```diff
@@ -0,0 +1,80 @@
+name: Create Update Issues
+
+on:
+  workflow_call:
+    inputs:
+      AUTH_APP_ID:
+        description: Github app id to authenticate in mobile and extention repositories
+        type: string
+        required: true
+    secrets:
+      AUTH_APP_PRIVATE_KEY:
+        description: Github app private key to authenticate in mobile and extention repositories
+        required: true
+
+jobs:
+  create-update-issues:
+    permissions:
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout head
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get release tag
+        id: releaseTag
+        run: |
+          echo "tag=$(git describe --tags --abbrev=0)" >> "$GITHUB_OUTPUT"
+        shell: bash
+
+      - name: Check released package
+        id: releaseCheck
+        env:
+          tag: ${{ steps.releaseTag.outputs.tag }}
+        run: |
+          if [[ "${tag}" == @metamask/*  ]] ; then
+            # Extract package name without the leading '@'
+            package_name="${tag#@}"
+            package_name="${package_name%@*}"
+            echo "package_name=${package_name}" >> "$GITHUB_OUTPUT"
+
+            # Extract version number
+            version="${tag##*@}"
+            echo "version=${version}" >> "$GITHUB_OUTPUT"
+
+            # Check if version number ends with .0.0
+            if [[ $version == *.0.0 ]]; then
+              echo "$package_name major update"
+              echo "major_version=true" >> "$GITHUB_OUTPUT"
+            else
+              echo "$package_name minor update"
+              echo "major_version=false" >> "$GITHUB_OUTPUT"
+            fi            
+          else
+            echo "Monorepo version update"
+            echo "major_version=false" >> "$GITHUB_OUTPUT"
+          fi
+        shell: bash
+
+      - name: Generate a token
```

Comment:
We use GitHup app here to avoid personal access token. PAT is connected to user and when that user will lost access to org also workload will stop working. On top off that is easier to rotate credentials for security team if required.

Reply:
We have a fairly well defined process for managing PATs, and we generate them using an account not tied to an individual (`metamaskbot`). We use them a lot, it would be quite onerous to create an app for each one.